.ci/pytorch/test.sh

```diff
@@ -242,15 +242,15 @@ test_dynamo_shard() {
 test_inductor_distributed() {
   # this runs on both single-gpu and multi-gpu instance. It should be smart about skipping tests that aren't supported
   # with if required # gpus aren't available
-  python test/run_test.py --include distributed/test_dynamo_distributed distributed/test_inductor_collectives --verbose
+  python test/run_test.py --continue-through-error --include distributed/test_dynamo_distributed distributed/test_inductor_collectives --verbose
   assert_git_not_dirty
 }
 
 test_inductor() {
   python tools/dynamo/verify_dynamo.py
-  python test/run_test.py --inductor --include test_modules test_ops test_ops_gradients test_torch --verbose
+  python test/run_test.py --continue-through-error --inductor --include test_modules test_ops test_ops_gradients test_torch --verbose
```

Comment:
```suggestion
  python test/run_test.py --inductor --include test_modules test_ops test_ops_gradients test_torch --verbose
```

Reply:
Removed `--continue-through-error` in https://github.com/pytorch/pytorch/pull/97988/commits/34795f70287defe82b5bc1198ee303a638049c18